### PR TITLE
Features

### PR DESCRIPTION
**Pull request by yay5a**
Hey all! I tried to work through getting the logic for the handlebars, to be able to fetch users data (consoles, games) and display it when they're logged in. I kinda dropped a refactor nuke in controllers/api, all while testing the server through insomnia and chrome. Mostly converting routes to arrow functions and implementing try/catch blocks. See comments below.

All in all, the server works and the site launches with cookies and everything. You can click on the links and it will take you to the all the pages, BUT only the nav bar at the top renders, and raw unfiltered HTML in the body instead of actual content. 

EDIT: I think l found it https://stackoverflow.com/questions/42751686/handlebars-template-being-rendered-as-raw-html

### CODE REVIEW

**Comment by yay5a** [comment]
I'm gonna keep trying to find a solution. But for now, I need to take a break. Been coding since 12AM

**Comment by Chartok** [comment]
I think I found it. Here: https://html.spec.whatwg.org/multipage/parsing.html#unexpected-markup-in-tables

Ok break time forreal. 
let me know if I'm at least on the right track xD

File: controllers/homeRoutes.js [hunk 1]
```diff
@@ -2,6 +2,51 @@
 const router = require('express').Router();
 const withAuthorization = require('../utils/auth');
 
+
```
Comment (yay5a) [inline controllers/homeRoutes.js:5]: moved routes to the top and array to the bottom; so the array isn't in our face when trying to look at routes.

File: package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "dateformat": "^5.0.3",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
-    "express-handlebars": "^7.0.7",
+    "express-handlebars": "^3.1.0",
```
Comment (yay5a) [inline package.json:27]: uninstalled current handlebars to a "allegedly" working previous version. still does not render.

File: utils/helper.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const handlebars = require("handlebars");
```
Comment (yay5a) [inline utils/helper.js:1]: wrote helper to handle the logic of users adding games and consoles

File: views/layouts/main.handlebars [hunk 4]
```diff
@@ -1,41 +1,54 @@
...
-        </ul>
-      </div>
-    </nav>
-    <div class="container">
-      {{{body}}}
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <a class="navbar-brand" href="/">GamerLi</a>
+    <div class="collapse navbar-collapse">
+      <ul class="navbar-nav ml-auto">
+        <li class="nav-item {{#if activeHome}}active{{/if}}">
```
Comment (yay5a) [inline views/layouts/main.handlebars:21]: {{#if active} {{/if}} was supposed to indicate if the link was clicked already by the user.

File: views/myLibrary.handlebars [hunk 5]
```diff
@@ -1,42 +1,25 @@
...
-        </div>
-        <div class="col-8">
-          <input
-            class="note-title"
-            placeholder="Note Title"
-            maxlength="28"
-            type="text"
-          />
-          <textarea class="note-textarea" placeholder="Note Text"></textarea>
-        </div>
```
Comment (yay5a) [inline views/myLibrary.handlebars:36]: Scrapped NoteTaker stencil

File: views/partials/myLibraryContent.handlebars [hunk 6]
```diff
@@ -0,0 +1,29 @@
+{{!-- views/myLibrary.handlebars --}}
```
Comment (yay5a) [inline views/partials/myLibraryContent.handlebars:1]: This is a partial that's supposed to template for reuse the user's library; may need some work or not needed.

**Pull request closed without merging**